consensus/ibft/ibft.go

```diff
@@ -299,6 +299,16 @@ func (i *Ibft) setupMechanism() error {
 		}
 	}
 
+	if i.bridge != nil {
+		bridgeMechanism, _ := BridgeFactory(i)
+
+		if err != nil {
+			return err
+		}
```

Comment:
You're not checking the BridgeFactory error here, but some leftover error variable

Reply:
Added at 59388a9e1e2f9a54baa081694c9dd5740f1b9928